storage/storage_helpers.hpp

```diff
@@ -13,7 +13,7 @@ namespace storage
 class CountryInfoGetter;
 class Storage;
 
-TMwmSize constexpr kMaxMwmSizeBytes = 100 /*Mb*/ * 1024 * 1024;
+TMwmSize constexpr kMaxMwmSizeBytes = 115 /*Mb*/ * 1024 * 1024;
```

Comment:
Err, this value is wrong. Please, re-read my comment. Why not to calculate this constant directly based on countries.txt?